app/main/views.py

```diff
@@ -1,15 +1,37 @@
 from flask import render_template
-from ..models import EditableHTML, IncidentReport
+from ..models import EditableHTML
```

Comment:
Note that there is a `from app.models` import below. You should from all the models imports together.